lib/components/checkboxes/checkbox_demo.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+
+import '../../resource/theme_color.dart';
+
+class CheckboxDemo extends StatefulWidget {
+  final bool isChecked;
+  final String label;
+  final ValueChanged<bool?>? onChanged;
+
+  const CheckboxDemo({
+    Key? key,
+    required this.isChecked,
+    required this.label,
+    this.onChanged,
+  }) : super(key: key);
+
+  @override
+  _CheckboxDemoState createState() => _CheckboxDemoState();
+}
+
+class _CheckboxDemoState extends State<CheckboxDemo> {
+  late bool isChecked;
```

Comment:
isChecked variable is initialized in the initState method therefore late keyword is unnecessary.